templates/README.md

```diff
@@ -0,0 +1,40 @@
+# gpuCI Script Templates
+
+## Overview
+
+These templates provide a guide for developers to assist them in adding the needed integration scripts to workwith the gpuCI. Every script here is required for full integration into the gpuCI infrastructure.
+
+## Usage
+
+The templates have been marked to show developers where to make changes on the files. These markers are made to be easily found using your preferred file search function. All comments with *exactly* two hashtags should be removed before being finalized.
+
+Marker | Meaning
+`##` | Generally represents useful information for guiding the user
+`##*` | Represents the *start* or *end* of an area in a script that is expected to be edited
+`## EDIT:` | A short description of what should be edited in the codeblock encapsulated by `##*`
+`|<` or `>|` | Represents a specific area in the code that should be edited. Also encapsulates a short description of what should replace it
+
+## Description
+
+* `rootbuild.sh` - Located at repository root `/`, this script is the main build script that builds your package from source. This script should be also be useable by users as a simplistic way to build the project. Rename this script to `build.sh` when committing to the repository.
```

Comment:
I think you should mirror the folder structure rather than put all of the files in the `templates/` dir since the directories matter.
So like `templates/ci/{cpu,gpu}/build.sh`.

Reply:
Good idea